### ISSUE TITLE
11GB GPU for training

### ISSUE THREAD

**Issue by MSiam** [a514514772/DISE-Domain-Invariant-Structure-Extraction#6]
Hello 
   I was wondering if there is a way that I can fit the model for training on a 11GB GPU without losing a lot in the segmentation accuracy. I initially tried to reduce the cropped size by half but that reduced the resulting segmentation accuracy by 17%. 

Thanks for your help

**Comment by hui-po-wang** [comment]
Hi,

If you want to reduce the crop size, I would suggest you also downsample the original image size so that the model can see enough context information. Besides, after applying the aforementioned settings, make sure that your model at the testing phase sees the images with the same resolution as those at the training phase.

Other possible solutions could be using smaller batch size, or making the reconstruction images smaller.

I would like to see any update from you If you have some further findings.

Thanks

**Comment by MSiam** [comment]
Thanks for your help I will try and see the final results

**Comment by MSiam** [comment]
Just to report for others working on 1GPU the only way to get it barely fit I used:
source_input_size= [675, 1200]
target_input_size= [480, 960]
cropped_size=[240,480]
batch_size=1

setting track_running_stats=True was what improved it since newer version of pytorch don have it by default. I got 37.1% mIoU on val set.
